retirement_api/utils/ss_utilities.py

```diff
@@ -11,22 +11,25 @@
 if you are under 22 years of age.</span> Please visit the \
 Social Security Administration's \
 <a href="http://www.ssa.gov/people/youngpeople/" \
-target="blank">advice page</a> for students and younger workers.\
+target="_blank" rel="noopener noreferrer">advice page</a> \
+for students and younger workers.\
 """
 TOO_YOUNG_ES = """\
 Lo sentimos. No podemos estimar sus beneficios si usted \
 es menor de 22 años de edad.</span> \
 Visite <a href="http://www.ssa.gov/people/youngpeople/" \
-target="_blank">la página</a> (en inglés) de la Administración \
+target="_blank" rel="noopener noreferrer">la página</a> \
+(en inglés) de la Administración \
 del Seguro Social para estudiantes y trabajadores jóvenes.\
 """
 TOO_OLD = """\
 <span class="h4">Sorry, our tool cannot provide an estimate because \
 your birthdate, {0}, means you are older than 70 and may already be receiving \
 benefits.</span> To check your benefits based on your actual \
 earnings record, contact the Social Security Administration or \
-open a <a href="http://www.socialsecurity.gov/myaccount/" target="_blank">\
-<em>my</em> Social Security</a> account.
+open a <a href="http://www.socialsecurity.gov/myaccount/" \
+target="_blank" rel="noopener noreferrer">\
+<em>my</em> Social Security</a> account.\
```

Comment:
I added a trailing `\` here as all other blocks of text in this file had it for the last line. Is this correct?

Reply:
Final backslash is no-op. At least our no-ops are consistent now!